### DISCUSSION TITLE
DBeaver prompts for driver download even if the driver is already present but only first time

### DISCUSSION THREAD

**Discussion by windowshome** [dbeaver/dbeaver#15048] [General]
DBeaver prompts for driver download even if the driver is already present (but only first time

**Comment by ShadelessFox** [comment]
Hello @windowshome,

This may happen when we update recommended driver version.

Perhaps the driver was indeed updated?

**Comment by markusb** [comment]
I have a similar problem. Ever time I start dbeaver I have to select the driver jar again. Then it works until the next new start of dbeaver.

How can I make it so that dBeaver remembers the driver across sessions ?
